sds_data_manager/lambda_code/SDSCode/create_schema.py

```
@@ -0,0 +1,95 @@
+import json
+import os
+
+import boto3
+import psycopg2
+import requests
+from psycopg2 import Error
+
+# TODO: The code in this lambda will be updated in the next PR
+# to reflect the schema that will be needed by the SDC.
+# Its current state is the bare minimum to get a table
+# created to allow files to be indexed.
+
+# Add code to get Secret instead
+s3 = boto3.client("s3")
+
+
+def send_response(event, context, response_data, response_status):
+    response_url = event["ResponseURL"]
+    response_body = {
+        "Status": response_status,
+        "Reason": "See the details in CloudWatch Log Stream: "
+        + context.log_stream_name,
+        "PhysicalResourceId": context.log_stream_name,
+        "StackId": event["StackId"],
+        "RequestId": event["RequestId"],
+        "LogicalResourceId": event["LogicalResourceId"],
+        "Data": response_data,
+    }
+
+    json_response_body = json.dumps(response_body)
+
+    headers = {"content-type": "", "content-length": str(len(json_response_body))}
+
+    try:
+        response = requests.put(response_url, data=json_response_body, headers=headers)
+        response.raise_for_status()
+    except Exception as e:
```

We shouldn't have a bare except here. Do we want to just let this error out instead of printing?